pylookyloo/api.py

```diff
@@ -25,6 +25,9 @@ def __init__(self, root_url: str='https://lookyloo.circl.lu/'):
         :param root_url: URL of the instance to query.
         '''
         self.root_url = root_url
+
+        if not urlparse(self.root_url)[0]:
```

Comment:
```suggestion
        if not urlparse(self.root_url).scheme:
```
That's more readable, isn't it?